app/src/main/res/layout/fragment_all_list.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
could you rename your xml files so that it has the role first? ex. admin_fragment_all_list, etc

Reply:
renamed